### PR TITLE
Adds detailed classifiers to setup.py

### PR DESCRIPTION
**Pull request by vatsala**
A possible fix for #403

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -98,7 +98,11 @@ def run(self):
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
```
Comment (ThomasWaldmann) [inline setup.py]: How about Python :: 2?

Comment (vatsala) [inline setup.py]: Oops, missed it. Adding that too.

**Comment by untitaker** [comment]
Could you use `git rebase -i` to squash all commits into one?

**Comment by vatsala** [comment]
@untitaker Done

**Comment by untitaker** [comment]
Excellent, thanks!

**Pull request merged**